src/examples/with-nested-dockerfile.yml

```diff
@@ -0,0 +1,15 @@
+description: >
+  Build/publish a Docker image building from a nested Dockerfile
+usage:
+  version: 2.1
+
+  orbs:
+    docker: circleci/docker@x.y.z
+
+  workflows:
+    build-docker-image-only:
+      jobs:
+        - docker/publish:
+            image: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
+            path: dockerfiles/prod
+            docker-context: ../..
```

Comment:
Please correct me if I'm wrong, but should `docker-context` be set to `.` instead?

I did a test of `docker build -f dockerfiles/prod/Dockerfile ../..` and it seemed to hang, while `docker build -f dockerfiles/prod/Dockerfile .` worked well.

Contents of the `dockerfiles/prod/Dockerfile` used in my test:
```
FROM ubuntu:latest
COPY readme .
```

Reply:
Hi, yes for a normal docker set up . Would work fine, but for the example I wanted to show I use case when you'd need to set the context rather than use default.

The example is trying to demonstrate having multiple dockerfiles in a nested directory but they include files 2 levels up in the context.